utils.py

```diff
@@ -328,3 +329,19 @@ def generate_data_frame_chunks(meta, chunk_size):
 
 def categorize_image(image, channel_axis=0):
     return np.argmax(image, axis=channel_axis)
+
+
+def coco_evaluation(output, image_ids, data_dir, dataset, category_ids):
```

Comment:
@taraspiotr I think that we should pass the filepath to annotation here so that this function can do just one thing. As of now if  depends on the `annotation.json` ground truth file. Maybe output should be dumped to filepath and then this evaluation will take 2 filepaths or we can load annotations before and pass loadded annotation lists for prediction and ground truth.

Reply:
@jakubczakon I think dumping prediction to file and than passing two filepaths will be better, because COCO class can only use filepath to initialize.

Reply:
@taraspiotr I see. In that case let's go with filepaths